crates/sc-proof-of-time/src/source.rs

```diff
@@ -23,19 +24,22 @@ pub struct PotSlotInfo {
     pub checkpoints: PotCheckpoints,
 }
 
+/// Proof of time slot information
+struct NewCheckpoints {
```

Comment:
This name does not sit well but I dont have better suggestions so non blocker

Reply:
I found a better name for it in https://github.com/subspace/subspace/pull/1922